src/frontend/js/hooks/useStepManager/index.spec.ts

```diff
@@ -0,0 +1,158 @@
+import { act } from '@testing-library/react';
+import { renderHook } from '@testing-library/react-hooks';
+import { Manifest, useStepManager } from '.';
+
+describe('useStepManager', () => {
+  it('reads the manifest', () => {
+    type Steps = 'step0' | 'step1';
+    const manifest: Manifest<Steps> = {
+      start: 'step1',
+      steps: {
+        step0: {
+          next: 'step1',
+        },
+        step1: {
+          next: 'step1',
+        },
```

Comment:
next of step0 and step1 is step1, is there something wrong no?

Reply:
Oh yes 😅